Bingo_CSD_412/Models/Category.cs

```diff
@@ -10,12 +10,62 @@ public class Category
         //static category TODO: have to implement in Database 
         public static readonly Category CSD_412_Category = new Category();
         public String Name { get; set; }
-        public String[] Words { get; set; }
+        public HashSet<string> Words { get; set; }
+        public int Id { get; }
```

Comment:
I think we don't need a separate Id field, Name should ensure category uniqueness by itself

Reply:
I have a couple of questions:
1. Are categories user defined?
2. Are we going to store user defined categories into our database?

If both questions are true, we may run into a problem if two separate users decide to create categories that have the same name. I proposed that we use a numeric identifier to highlight uniqueness. When a user creates a new category, we could return to them the numeric identifier so that they can retrieve the category they wanted in the case that there are many categories of the same name. Otherwise we’ll have to add input validation logic to confirm that name property is unique.

Reply:
I guess we can have common categories that are shared by all the users. What do you think?

Reply:
I think that'll work for the scope of this project!